operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2053,6 +2086,326 @@ gateway:
         hosts:
           - gateway.gem.example.com
 
+graphite:
+  # -- If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
+  enabled: false
+
+  querier:
+    replicas: 2
+
+    schemasConfiguration:
+      storageSchemas: |-
+        [default]
+        pattern = .*
+        intervals = 0:1s
+        retentions = 10s:8d,10min:1y
+      storageAggregations: |-
+        [default]
+        aggregationMethod = avg
+        pattern = .*
+        xFilesFactor = 0.1
+
+    service:
+      annotations: {}
+      labels: {}
+
+    # -- Resources for graphite-querier pods
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+
+    # -- Additional graphite-querier container arguments, e.g. log level (debug, info, warn, error)
+    extraArgs: {}
+    # -- The name of the PriorityClass for graphite-querier pods
+    priorityClassName: null
+    # -- Labels for graphite-querier pods
+    podLabels: {}
+    # -- Annotations for graphite-querier pods
+    podAnnotations: {}
+
+    nodeSelector: {}
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: target
```

target is an outdated label, so this affinity rule doesn't match anything you can use app.kubernetes.io/component

```suggestion
                  - key: app.kubernetes.io/component
```